### PR TITLE
Enables per-bundle configuration of auto build 

### PR DESCRIPTION
**Pull request by aluuu**
See #389

### CODE REVIEW

**Comment by aluuu** [comment]
@miracle2k, looks like build failed because of unavailability of file `http://michael.elsdoerfer.name/rjsmin/rjsmin-1.0.1-webassets.tar.gz`.

File: src/webassets/bundle.py [hunk 1]
```diff
@@ -741,7 +744,7 @@ def _urls(self, ctx, extra_filters, *args, **kwargs):
             # With ``auto_build``, build the bundle to make sure the output is
             # up to date; otherwise, we just assume the file already exists.
             # (not wasting any IO ops)
-            if ctx.auto_build:
+            if ctx.auto_build or self.auto_build:
```
Comment (miracle2k) [inline src/webassets/bundle.py]: ``ctx.auto_build`` should already consider ``self._config['auto_build']``. This is not the case in your experience?

Comment (miracle2k) [inline src/webassets/bundle.py]: You can see this in lines like this ``ctx = wrap(self.env, self)``.

**Comment by aluuu** [comment]
Oh, I got it. 

I'll close related issue as resolved.

**Pull request closed without merging**